tutorials/single_node_tutorial/single_gpu_tutorial.ipynb

```diff
@@ -0,0 +1,3806 @@
+    "\n",
+    "NeMo Curator includes the following modules to perform data curation:\n",
+    "- Data download and Extraction\n",
+    "- Language identification and separation\n",
+    "- Text reformatting and cleaning\n",
+    "- Quality filtering\n",
+    "- Document-level deduplication\n",
+    "- Multilingual downstream-task decontamination\n",
+    "- Distributed Data Classification\n",
+    "- Personal identifiable information (PII) redaction\n",
+    "\n",
+    "NeMo Curator team has perform ablation experiments using Common Crawl dataset to train a 357M GPT-style model to assess the effect of different curation stage on model performance. \n",
+    "\n",
+    "![alt text](./image/zeroshot_ablations.png)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "58d062aa",
+   "metadata": {},
+   "source": [
+    "## About this notebook\n",
+    "\n",
+    "\n",
+    "This notebook will use **Thai Wikipedia dataset** as example to demonstrate a typical data curation pipeline using NeMo Curator. After running through this script, user will be able to know how to use NDC to download wikipedia data, perform language separation using fasttext, perform GPU based exact deduplication and fuzzy deduplication and use CPU based heuristic filtering. \n",
+    "\n",
+    "Step description:\n",
+    "1. Download and extract data\n",
+    "2. Language detection and separation\n",
+    "3. GPU based deduplication\n",
+    "    1. Exact deduplication\n",
+    "    2. Fuzzy deduplication\n",
+    "4. Heuristic filtering\n",
+    "\n",
+    "What is not included:\n",
+    "1. Customized downloading\n",
+    "2. Classifier filtering\n",
+    "3. Downstream-task decontamination\n",
+    "4. Distributed data classification with PyTorch models\n",
+    "5. Personal identifiable information (PII) redaction\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a6e3492e",
+   "metadata": {},
+   "source": [
+    "## Prerequisites\n",
+    "\n",
+    "### System Requirements\n",
+    "Here is the hardware setting for this notebook\n",
+    "\n",
+    "**GPU**: NVIDIA A10 24G. \n",
+    "\n",
+    "**CUDA & Nvidia Drivers**: CUDA 12.2 with Driver 535.154.05\n",
+    "\n",
+    "**OS**: ubuntu 22.04\n",
+    "\n",
+    "### Getting NeMo Framework Training Container\n",
+    "- Get access to the container via https://developer.nvidia.com/nemo-framework\n",
+    "- Set your docker credentials \n",
+    "    ```bash\n",
+    "    docker login nvcr.io\n",
+    "\n",
+    "    Username: $oauthtoken\n",
+    "    Password: <Your NGC Key>\n",
+    "- Get NeMo NeMo Framework Training Container\n",
+    "    ```bash\n",
+    "    docker pull nvcr.io/ea-bignlp/ga-participants/nemofw-training:24.01\n"
```

Comment:
I'd either change this to the dev container or the 24.05 tag (which hasn't been released yet).
```
docker pull nvcr.io/nvidia/nemo:dev.framework
docker pull nvcr.io/nvidia/nemo:24.05.framework
```
Since the other container versions don't have the latest version of NeMo Curator that the tutorial uses.

Reply:
Fixed